crates/hir-ty/src/tests/macros.rs

```diff
@@ -335,16 +338,16 @@ fn recurisve_macro_expanded_in_stmts() {
         }
         "#,
         expect![[r#"
-            !0..7 'leta=3;': {unknown}
-            !0..7 'leta=3;': {unknown}
-            !0..13 'ng!{[leta=3]}': {unknown}
-            !0..13 'ng!{[leta=]3}': {unknown}
-            !0..13 'ng!{[leta]=3}': {unknown}
+            !0..7 'leta=3;': ()
+            !0..13 'ng!{[leta=3]}': ()
+            !0..13 'ng!{[leta=]3}': ()
+            !0..13 'ng!{[leta]=3}': ()
+            !0..13 'ng!{[let]a=3}': ()
             !3..4 'a': i32
             !5..6 '3': i32
             196..237 '{     ...= a; }': ()
-            229..230 'b': i32
-            233..234 'a': i32
+            229..230 'b': {unknown}
+            233..234 'a': {unknown}
```

Comment:
Unsure why this is happening now 😕 All the other tests changes seem correct to me, but this one obviously isn't